### ISSUE TITLE
cli: error in cmd_run.py with python3

### ISSUE THREAD

**Issue by ankan17** [systemslab/popper#430]
Here is a traceback of the error:
```bash
$ popper run --no-badge-update
Traceback (most recent call last):
  File "/tmp/popper-env/bin/popper", line 11, in <module>
    sys.exit(cli())
  File "/tmp/popper-env/lib/python3.4/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/tmp/popper-env/lib/python3.4/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/tmp/popper-env/lib/python3.4/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/tmp/popper-env/lib/python3.4/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmp/popper-env/lib/python3.4/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/tmp/popper-env/lib/python3.4/site-packages/click/decorators.py", line 64, in new_func
    return ctx.invoke(f, obj, *args[1:], **kwargs)
  File "/tmp/popper-env/lib/python3.4/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/tmp/popper-env/lib/python3.4/site-packages/popper/commands/cmd_run.py", line 87, in cli
    pipes_from_log = pipelines_from_commit_message(project_pipelines)
  File "/tmp/popper-env/lib/python3.4/site-packages/popper/commands/cmd_run.py", line 166, in pipelines_from_commit_message
    if "Merge" in msg:
TypeError: 'str' does not support the buffer interface
```

**Comment by Franko1307** [comment]
I think this issue is related with https://github.com/systemslab/popper/issues/425

**Comment by ankan17** [comment]
@Franko1307 Thanks for pointing it out

**Comment by eltrufas** [comment]
This got fixed in #426.
